server/fleet/activities.go

```diff
@@ -0,0 +1,39 @@
+package fleet
+
+import (
+	"context"
+	"encoding/json"
+)
+
+const (
+	CreatedPackActivityType               = "created_pack"
+	EditedPackActivityType                = "edited_pack"
+	DeletedPackActivityType               = "deleted_pack"
+	AppliedSpecPackActivityType           = "applied_spec_pack"
+	CreatedSavedQueryActivityType         = "created_saved_query"
+	EditedSavedQueryActivityType          = "edited_saved_query"
+	DeletedSavedQueryActivityType         = "deleted_saved_query"
+	DeletedMultipleSavedQueryActivityType = "deleted_multiple_saved_query"
+	AppliedSpecSavedQueryActivityType     = "applied_spec_saved_query"
+	CreatedTeamActivityType               = "created_team"
+	DeletedTeamActivityType               = "deleted_team"
+	LiveQueryActivityType                 = "live_query"
```

Comment:
Nit: Can we prefer naming constants with the constant "type" first?

eg. `ActivityTypeCreatedPack`

This is consistent with how we've tried to name other constants, and I think the "namespacing" makes it easier to tell their purpose.